simulator/src/main/java/com/hazelcast/simulator/worker/WorkerCommandRequestProcessor.java

```diff
@@ -100,10 +100,9 @@ public void run() {
             while (running) {
                 try {
                     CommandRequest request = requestQueue.take();
-                    if (request == null) {
-                        throw new NullPointerException("request can't be null");
+                    if (request != null) {
```

Comment:
My goal for this PR was to increase code coverage, so I was very conservative with changing any behavior. And (sometimes) I'm just a bit cautious when chaning something, especially before removing checks or try/catch-blocks. It's a "someone put it there for a reason I don't know" feeling.